src/styles/components/_form.scss

```diff
@@ -17,6 +17,10 @@ form {
 
   box-shadow: none;
 
+  .hidden {
```

Comment:
Is there an existing class called `d-none` on the donation thank you pages with the same CSS property?

Reply:
Whoops! I think what happened was I forgot to clean out the branch from the unsubscribe changes, and so those commits got put into the new pull request.  I removed the ".hidden" style since it's not necessary.